crontab

```diff
@@ -0,0 +1 @@
+* * * * * php /var/www/artisan schedule:run >> /var/log/cron.log 2>&1
```

Comment:
Why did we need to change cron to run from supervisord?  There already exists a mechanism to run this in the entrypoint for the cron service.  Also, the Dockerfile for cid and coci copy in their own copies of the crontab.  Take a look at docker/front/cron/ and docker/front/Dockerfile in each of those projects. eg:
`COPY docker/front/cron/scheduler.sh /bin/scheduler.sh
COPY docker/front/cron/root /var/spool/cron/crontabs/root
RUN chmod +x /bin/scheduler.sh`